control_panel_api/tests/test_authentication.py

```diff
@@ -0,0 +1,52 @@
+from django.test import override_settings
+import jwt
+from model_mommy import mommy
+from rest_framework.reverse import reverse
+from rest_framework.status import HTTP_403_FORBIDDEN, HTTP_200_OK
+from rest_framework.test import APITestCase
+
+from control_panel_api.authentication import Auth0JWTAuthentication
+
+
+# Decode token at https://jwt.io/
+GOOD_TOKEN = 'eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJlbWFpbCI6ImphbWVzQGptb3ouY28udWsiLCJuYW1lIjoiSmFtZXMgTW9ycmlzIiwiYXVkIjoiYXVkaWVuY2UiLCJzdWIiOiJnaXRodWJ8MTIzNDUifQ.Vfvhtm_TXbtOBKcIWed0YzVH7gIKlSdzg36bbIK6UZ4'  # noqa
```

Comment:
We could improve the token tests and make the more readable by creating a function to encode a token

Reply:
Yep, I will update the test to build the token with `jwt.encode`, so we can see what is inside.